src/features/profile/profile_section.ts

```diff
@@ -39,7 +40,8 @@ export function getProfileSection(options: ProfileOptions): AnyHomeTabBlock[] {
           text:
             `*👤 Status* · ${options.status}\n` +
             `*⭐️ Community Credits* · ${options.communityCredits}/6\n` +
-            "*🏆 Liga* · Credit Warrior",
+            `*🥇 Rank* · ${options.rank}\n` +
```

Comment:
Can we also add this to the leaderboard like:

```
1. <scholar1>
2. <scholar2>
3. <scholar4>
...
<self.rank> <self>
```

And we could also make the self row bold.